optaweb-vehicle-routing-backend/src/main/resources/application.properties

```diff
@@ -21,7 +21,7 @@ app.routing.osm-file=belgium-latest.osm.pbf
 
 # Logging
 logging.level.root=info
-logging.level.org.optaplanner.core=info
+logging.level.org.optaplanner.core=debug
```

Comment:
Please revert, the application is not supposed to log on debug level in production. Logging level in local environment can be customized with `application-local.properties`. You can start with an empty file and override any properties you like. The file is ignored so you don't need to worry about having uncommitted changes or committing it unintentionally.